code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -86,7 +86,7 @@
 	//back
 	if(back && back.is_visible())
 		if(back.blood_DNA && back.blood_DNA.len)
-			msg += "<span class='warning'>[t_He] [t_has] [bicon(back)] [back.a_stained()] [back] on [t_his] back! [format_examine(back, "Examine")][back.description_accessories()]</span>\n"
+			msg += "<span class='warning'>[t_He] [t_has] [bicon(back)] [back.a_stained()] [back.name] on [t_his] back! [format_examine(back, "Examine")][back.description_accessories()]</span>\n"
```

Comment:
\the back is more flexible and accomplishes the same thing

Reply:
In this case using the  `[back]` makes it say "a blood-stained the backpack" so I was thinking to avoid that.

Reply:
it's the \the macro that's doing the heavy lifting on this one

Reply:
`[back.a_stained()]` returns `"a blood-stained"`, so the article is already before the adjective. Is there a good way to use the inbuilt `\the` or `\an` macro to splice the adjective "blood-stained" in between "a" and "backpack"?

It seems like most of the other slots already used `.name` but there were just a few (ears, eyes, and back) that used the datum itself.